src/serac/infrastructure/accelerator.hpp

```diff
@@ -15,17 +15,47 @@
 
 #if defined(__CUDACC__)
 #define SERAC_HOST_DEVICE __host__ __device__
+#define SERAC_HOST __host__
+#define SERAC_DEVICE __device__
+
+#if __CUDAVER__ >= 75000
+#define SERAC_SUPPRESS_NVCC_HOSTDEVICE_WARNING #pragma nv_exec_check_disable
```

Comment:
Can we add a snippet (either here or where this gets used) explaining why we need to suppress this warning?

Reply:
@joshessman-llnl @samuelpmishLLNL Can you mention why this suppression is needed? Thanks!

Reply:
see: https://godbolt.org/z/9M57xao1q
nvcc will sometimes emit a warning if a `__host__ __device__` function calls a `__host__`-only or `__device__`-only function. `make_tensor` is marked `__host__ __device__` and is used frequently in the code base, so it was emitting a lot of warnings. This #pragma directive suppresses the warning for a specific function.